assets/chat/js/formatters.js

```diff
@@ -374,6 +374,7 @@ class UrlFormatter {
                 url = self.encodeUrl(m[0]);
                 const extra = self.encodeUrl(decodedUrl.substring(m[0].length));
                 const href = scheme + url;
+                url = href.replace(/(^\w+:|^|www.)|\/\//g, ''); // Removes https://www. from links in chat
```

Comment:
I am not interested in this feature. Others may have interest in it though.

Reply:
Pardon my unfamiliarity with GitHub, "requested changes" is akin to waiting for other's feedback and not for me to remove the code so you can merge it the code you want, ye?

Reply:
You are fine. I am waiting for others that have been requested to take a look and decide. After they review the PR then we can merge it in. If they do decide this feature is unwanted coupled with the modifier then we will ask you to remove it then we will merge. :+1: